### DISCUSSION TITLE
Include Transaction ID for Spent UTXO

### DISCUSSION THREAD

**Discussion by theeldermillenial** [CardanoSolutions/kupo#182] [Archives]
### What is your idea? Provide a use case.

There is currently a field to describe the block time a UTxO was spent at. It would be helpful to include the transaction id that the UTxO was spent in.

Many protocols on Cardano use some type of batching system, meaning a number of operations take two transactions to complete. This is especially true for DEX operations.

### Why is it a good idea?

Being able to easily link a UTxO to the transaction it is spent in would make li ling these operations much simpler.

### What is the current alternative and why is it not good enough?

I think there are a few alternatives to this, but the alternative I would currently use would be to query dbsync to get the transaction id from the tx_out table. I'm sure it might be possible to do this in ogmios as well.

**Comment by KtorZ** [comment]
Somewhat related to: https://github.com/CardanoSolutions/kupo/discussions/163

**Reply by theeldermillenial** [reply]
So it is! I did try to search through old issues, but that didn't immediately register. Yes, I think these things are highly related.

Any chance of this or 163 being implemented? I don't know how big of a lift it might be.

**Reply by KtorZ** [reply]
Coming with 2.10.0 tomorrow.
